config/webpack.config.cjs

```diff
@@ -258,6 +258,7 @@ module.exports = function (webpackEnv) {
               // Pending further investigation:
               // https://github.com/terser-js/terser/issues/120
               inline: 2,
+              drop_console: true,
```

Comment:
main change. since eslint is no longer banning console logs, webpack will drop it before shipping to prod

Reply:
oh interesting, does this only drop console.logs or other consoles like error/warn

Reply:
warn wasn't an option, but noticing in my code that it underlines it anyway (without erroring)

Reply:
looks like it drops everything. should we keep some stuff around?

![image](https://user-images.githubusercontent.com/12162433/142717595-e4f1d1e2-b94f-4af2-9cc7-7489bae7a9c0.png)

Reply:
![image](https://user-images.githubusercontent.com/12162433/142717620-4c36dc7b-0a80-4448-960f-43d3c364dc67.png)